### ISSUE TITLE
No signal above threshold, still finds spikes

### ISSUE THREAD

**Issue by goldenbuddy** [csn-le/wave_clus#196]
Hi Fernando,

We have discovered an issue in our lab where there is no signal above the threshold value yet the program still detects spikes. Across three different recordings, it detected the exact same number of spikes when no signal was above the threshold. I was wondering if you had any insight on this? I have pasted the images below each from a five second recording of the same population of neurons. My first instincts were possibly a down sampling issue but I do not think that is the case. The default parameters for the program were used. Let me know if you need any more information!

Thank you so much! 
- Sam

![image](https://user-images.githubusercontent.com/67498049/122275864-08ed0000-ce99-11eb-9d30-3e7c7174168f.png)
![image](https://user-images.githubusercontent.com/67498049/122275903-12766800-ce99-11eb-9b89-86ce19704352.png)
![image](https://user-images.githubusercontent.com/67498049/122275924-1a360c80-ce99-11eb-8e2b-1c30ec907006.png)

**Comment by ferchaure** [comment]
Hi Sam, how are you running/opening those sortings? 

When less than 16 spikes were detected you can get a "Not enough spikes in the file" warning or a similar error using the GUI. It looks like the gui is finding another _times_ file and opening that.
Maybe all the raw files have the same name, and you have a _times_thatname.mat_ in your path?

**Comment by goldenbuddy** [comment]
Hi Fernando,

Sorry for the delayed reply! So I created a program that would automatically handle directories to avoid this problem and keep the data organized. This program also used the batch files. We found that if there weren't enough spikes in a file, the program would look for another file along the path with the same name and reference that one for clustering. I had a few other folders containing other wave clus outputs along the path accidentally and it referenced those causing the problem I had above.

Thanks for your time!
-Sam
